src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
+`publish`, `readme`, `repository`, `rust-version`, and`version`.
+
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. Inheriting dependencies was added as a
+way to reduce duplication of dependencies across members of a workspace.
+Inheriting a dependency is as simple as setting the dependency in the workspace
+under `[workspace.dependencies]` as you would in`[dependencies]`. From there
+[specify that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
+Full example:
+
+```toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
+[workspace.dependencies]
+log = "0.3.1"
+serde = { git = 'https://github.com/serde-rs/serde' }
+wasm-bindgen-cli = { path = "crates/cli" }
+```
+
+```toml
+# [PROGECT_DIR]/bar/Cargo.toml
+[project]
+name = "bar"
+version = "0.2.0"
+
+[dependencies]
+log.workspace = true
+serde.workspace = true
+wasm-bindgen-cli.workspace = true
+```
+
+Dependencies in `[workspace.dependencies]` cannot be declared as `optional`.
+The `optional` key must be omitted or, if present, must be `false`. Dependencies
+in the `[dependencies]`, `[dev-dependencies]`, `[build-dependencies]`, and
+`[target."...".dependencies]` sections support the ability to reference the
+`[workspace.dependencies]` definition of dependencies.
+
+[inheriting-a-dependency-from-a-workspace]: #inheriting-a-dependency-from-a-workspace
+[workspaces]: workspaces.md
+[the-workspacemetadata-table]: workspaces.md#the-workspacemetadata-table
+
+### Inheriting a dependency from a workspace
+
+This would be in [`specifying-dependencies.md`][specifying-dependencies],
+under [Renaming dependencies in Cargo.toml][renaming-dependencies-in-cargotoml]
+
+Dependencies can be inherited from a workspace. To do this, specify the
+dependency in the workspace's [`[workspace.dependencies]`][workspace.dependencies] table.
+After that add it to the `[dependencies]` table with `dep.workspace = true`.
+
+The `workspace` key cannot be defined with other keys that configure the source of the
+dependency. This means you cannot define `workspace` with keys like `version`, `registry`,
+`registry-index`, `path`, `git`, `branch`, `tag`, `rev`, or `package`. The `workspace`
+key can be combined with other keys, however:
+
+- `optional` - this introduces an optional dependency as usual, as well as a feature named
+  after the key (left hand side) of the dependency directive). Note that the
+  `[workspace.dependencies]` table is not allowed to specify `optional`.
+
+- `features` - this indicates, as usual, that extra features are being enabled over
+  the already-enabled features in the directive found in `[workspace.dependencies]`. The
+  result set of enabled features is the union of the features specified inline with
+  the features specified in the directive in the workspace table.
```

Comment:
I feel like this could be more direct to help people more quickly see the pertinent information about this.

Maybe
```md
- `features`: These are additive with the features declared in the `[workspace.dependencies]`, see [...link to features...]
```

Reply:
That is much less verbose but gets the point across well, I like it